internal/os/os.go

```diff
@@ -103,14 +103,15 @@ func (o *OS) Init(config models.DeviceConfigurationMessage) error {
 }
 
 func (o *OS) Update(configuration models.DeviceConfigurationMessage) error {
+	newOSInfo := configuration.Configuration.Os
```

Comment:
Should we implement this on Init() observer and skip the Update here?

Reply:
I think it should be as it is - we want to protect `Update` from `nil` OS config in any case and don't report error because of that - not only for `Init`